elixir/lib/homework_web/resolvers/transactions_resolver.ex

```diff
@@ -28,7 +32,7 @@ defmodule HomeworkWeb.Resolvers.TransactionsResolver do
   Create a new transaction
   """
   def create_transaction(_root, args, _info) do
-    case Transactions.create_transaction(args) do
+    case Transactions.create_transaction(transaction_amount_to_cents(args)) do
```

Comment:
I wasn't sure if I should call it this way, or pipe it, but left it this way for the sake of speed